unity-client/Assets/Rendering/Culling/CullingControllerUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+namespace DCL.Rendering
+{
+    public static class CullingControllerUtils
+    {
+        public static void DrawBounds(Bounds b, Color color, float delay = 0)
```

Comment:
How do you use this debug method? Should we wrap it in a `UNITY_EDITOR` directive?

Reply:
The caller is already wrapping it in `UNITY_EDITOR`.